### DISCUSSION TITLE
Loading the software

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/GoOdf#52] [Ideas]
It would be useful to allow one to be able choose between loading last file or starting a new one.

**Comment by Bumblebee001** [comment]
Also one may wish to save file as another &/or in a different directory.

**Comment by larspalo** [comment]
When, in the future, some kind of stored settings will be implemented recently used files might be added as well as custom startup directory.

Save as... is much easier than you know! Just change the name of the .organ file in the [Organ] section, it should even be possible to alter the directory but that's more experimental as all paths must then be changed (and that may or may not work reliably yet)... The save as behavior is well documented in the readme file, wiki and the screencasts. I consider it working exactly as it should already.

**Comment by Bumblebee001** [comment]
Thanks for the tip! I didn’t think if it.

**Comment by ahall41** [comment]
Another suggestion - it would be really useful if you could invoke GOODF for a file from File Manager (right click -> Open with GOODF). At the moment it launches GOODF but doesn't load the file.
